src/swap/SwapReviewScreen.tsx

```diff
@@ -0,0 +1,331 @@
+                  style={styles.tokenText}
+                  amount={divideByWei(swapInfo?.unvalidatedSwapTransaction?.sellAmount)}
+                  tokenAddress={fromToken}
+                  showLocalAmount={false}
+                  testID={'FromSwapAmountToken'}
+                />
+                <TokenDisplay
+                  style={styles.tokenSubText}
+                  amount={divideByWei(swapInfo?.unvalidatedSwapTransaction?.sellAmount)}
+                  tokenAddress={fromToken}
+                  showLocalAmount={true}
+                  testID={`FromSwapAmountTokenLocal`}
+                />
+              </View>
+            </View>
+            <View style={styles.tallRow}>
+              <View style={styles.column}>
+                <Text style={styles.label}>{t('swapReviewScreen.swapTo')}</Text>
+              </View>
+              <View style={[styles.column, styles.tokenDisplayView]}>
+                <TokenDisplay
+                  style={[styles.tokenText, { color: colors.greenUI }]}
+                  amount={divideByWei(
+                    swapInfo?.unvalidatedSwapTransaction?.buyAmount -
+                      swapInfo?.unvalidatedSwapTransaction?.gas
+                  )}
+                  tokenAddress={toToken}
+                  showLocalAmount={false}
+                  testID={'ToSwapAmountToken'}
+                />
+                <TouchableOpacity
+                  style={styles.touchableRow}
+                  onPress={() => setEstimatedDialogVisible(true)}
+                  hitSlop={variables.iconHitslop}
+                >
+                  <Text style={[styles.tokenSubText, { marginRight: 4 }]}>
+                    {t('swapReviewScreen.estimatedAmountTitle')}
+                  </Text>
+                  <InfoIcon size={12} color={colors.gray4} />
+                </TouchableOpacity>
+              </View>
+            </View>
+            <View style={styles.separator} />
+          </View>
+          <View style={styles.subContentContainer}>
+            <Text style={styles.transactionDetailsLabel}>
+              {t('swapReviewScreen.transactionDetails')}
+            </Text>
+            <View style={styles.row}>
+              <View style={styles.column}>
+                <Text style={styles.label}>{t('exchangeRate')}</Text>
+              </View>
+              <View style={styles.column}>
+                <Text style={styles.transactionDetailsRightText}>
+                  {`1 ${fromTokenSymbol} ≈ ${formatValueToDisplay(
+                    new BigNumber(swapInfo?.unvalidatedSwapTransaction?.price)
+                  )} ${toTokenSymbol}`}
+                </Text>
+              </View>
+            </View>
+            <View style={styles.row}>
+              <View style={styles.column}>
+                <Text style={styles.label}>{t('swapReviewScreen.estimatedGas')}</Text>
+              </View>
+              <View style={[styles.column, styles.tokenDisplayView]}>
```

Comment:
i think this View is not needed

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2767/commits/70960e345f0debe2cbefb3fce5e70ef19e3fddca!